samples/matter/light_bulb/CMakeLists.txt

```diff
@@ -12,7 +12,12 @@ get_filename_component(MATTER_MODULE_ROOT $ENV{ZEPHYR_BASE}/../modules/lib/matte
 set(mcuboot_KCONFIG_ROOT ${MATTER_MODULE_ROOT}/config/nrfconnect/chip-module/Kconfig.mcuboot.root)
 set(multiprotocol_rpmsg_KCONFIG_ROOT ${MATTER_MODULE_ROOT}/config/nrfconnect/chip-module/Kconfig.multiprotocol_rpmsg.root)
 
-if(NOT CONF_FILE STREQUAL "prj_no_dfu.conf")
+# For prj.conf the CONF_FILE is empty. In other case extract the exact file name from the path string.
+if(CONF_FILE)
+    string(REGEX MATCH "\\prj[^ $]+\\.conf" CONFIG_FILE_NAME ${CONF_FILE})
```

Comment:
Wouldn't `get_filename_component(... NAME)` be cleaner?

Reply:
Or sth like:
`cmake_path(GET <path-var> FILENAME <out-var>)`

Haven't tested it though.

Reply:
Right, I overcomplicated it a little bit.